apps/netsuite/tasks.py

```
@@ -1553,29 +1565,35 @@ def get_valid_reimbursement_ids(reimbursement_ids: List, platform: PlatformConne
 
-        if all_expense_paid:
-            payloads.append({'id': report_id, 'paid_notify_at': datetime.now(timezone.utc).strftime('%Y-%m-%dT%H:%M:%S.%fZ')})
-            reports_to_be_marked.add(report_id)
+        if payloads:
+            mark_paid_on_fyle(platform, payloads, reports_to_be_marked, workspace_id)
+    except (InvalidTokenError, InternalServerError) as e:
+        logger.info('Invalid Fyle refresh token or internal server error for workspace %s: %s', workspace_id, str(e))
 
-    if payloads:
-        mark_paid_on_fyle(platform, payloads, reports_to_be_marked, workspace_id)
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)